src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbol.cs

```diff
@@ -1372,7 +1165,7 @@ internal override void PostDecodeWellKnownAttributes(ImmutableArray<CSharpAttrib
                 Debug.Assert(_lazyCustomAttributesBag != null);
                 Debug.Assert(_lazyCustomAttributesBag.IsDecodedWellKnownAttributeDataComputed);
 
-                if (_containingType.IsComImport && _containingType.TypeKind == TypeKind.Class)
+                if (ContainingType.IsComImport && ContainingType.TypeKind == TypeKind.Class)
```

Comment:
Consider caching and reusing ```ContainingType```. This code is valid only for member methods, not for local functions.